test/python/transpiler/test_preset_passmanagers.py

```diff
@@ -825,21 +841,21 @@ def test_layout_tokyo_2845(self, level):
             2: Qubit(QuantumRegister(15, "ancilla"), 2),
             3: Qubit(QuantumRegister(15, "ancilla"), 3),
             4: Qubit(QuantumRegister(15, "ancilla"), 4),
-            5: Qubit(QuantumRegister(15, "ancilla"), 5),
-            6: Qubit(QuantumRegister(15, "ancilla"), 6),
-            7: Qubit(QuantumRegister(15, "ancilla"), 7),
-            8: Qubit(QuantumRegister(3, "qr1"), 1),
-            9: Qubit(QuantumRegister(15, "ancilla"), 8),
-            10: Qubit(QuantumRegister(15, "ancilla"), 9),
-            11: Qubit(QuantumRegister(15, "ancilla"), 10),
-            12: Qubit(QuantumRegister(3, "qr1"), 0),
-            13: Qubit(QuantumRegister(3, "qr1"), 2),
-            14: Qubit(QuantumRegister(2, "qr2"), 1),
-            15: Qubit(QuantumRegister(15, "ancilla"), 11),
-            16: Qubit(QuantumRegister(15, "ancilla"), 12),
-            17: Qubit(QuantumRegister(15, "ancilla"), 13),
-            18: Qubit(QuantumRegister(15, "ancilla"), 14),
-            19: Qubit(QuantumRegister(2, "qr2"), 0),
+            5: Qubit(QuantumRegister(3, "qr1"), 2),
+            6: Qubit(QuantumRegister(2, "qr2"), 0),
+            7: Qubit(QuantumRegister(2, "qr2"), 1),
+            8: Qubit(QuantumRegister(15, "ancilla"), 5),
+            9: Qubit(QuantumRegister(15, "ancilla"), 6),
+            10: Qubit(QuantumRegister(3, "qr1"), 1),
+            11: Qubit(QuantumRegister(3, "qr1"), 0),
+            12: Qubit(QuantumRegister(15, "ancilla"), 7),
+            13: Qubit(QuantumRegister(15, "ancilla"), 8),
+            14: Qubit(QuantumRegister(15, "ancilla"), 9),
+            15: Qubit(QuantumRegister(15, "ancilla"), 10),
+            16: Qubit(QuantumRegister(15, "ancilla"), 11),
+            17: Qubit(QuantumRegister(15, "ancilla"), 12),
+            18: Qubit(QuantumRegister(15, "ancilla"), 13),
+            19: Qubit(QuantumRegister(15, "ancilla"), 14),
```

Comment:
Is there a reason why we don't call each register entry as, for example, `ancilla[2]` instead of `Qubit(QuantumRegister(15, "ancilla"), 2)`?

Reply:
Apologies, I have cited the wrong lines in this case. Yes, what you said makes sense, I understand the mapping which is what the dict is showing here.

The subject to my question is more visible here:

https://github.com/ElePT/qiskit/blob/26f3508922fb07bb8da868eb7f39bbf8b40e08ff/test/python/transpiler/test_preset_passmanagers.py#L917-L938

My question was more about the way the dict that's being compared is initialized. Since we already had registers `qr` and `ancilla` containing qubits identical to the ones we're initializing in the dict, we could instead refer to these in the `sabre_layout` mappings. For example, instead of doing:

```py
sabre_layout = {
            0: Qubit(QuantumRegister(15, "ancilla"), 0),
            1: Qubit(QuantumRegister(15, "ancilla"), 1),
            2: Qubit(QuantumRegister(15, "ancilla"), 2),
            3: Qubit(QuantumRegister(15, "ancilla"), 3),
            4: Qubit(QuantumRegister(15, "ancilla"), 4),
            5: Qubit(QuantumRegister(5, "qr"), 1),
            6: Qubit(QuantumRegister(5, "qr"), 0),
            7: Qubit(QuantumRegister(5, "qr"), 4),
            ...
```

We could do:
```py
sabre_layout = {
            0: ancilla[0],
            1: ancilla[1],
            2: ancilla[2],
            3: ancilla[3],
            4: ancilla[4],
            5: qr[1],
            6: qr[0],
            7: qr[4],
...
```
And it would result in the same outcome.

Is there was a reason for this not being done?

Reply:
Yes, there's no problem with that. As I said it was a trivial change. It all LGTM!